packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -17,8 +17,13 @@ import 'theme_data.dart';
 /// such an image, the user's initials. A given user's initials should
 /// always be paired with the same background color, for consistency.
 ///
+/// If [foregroundImage] fails then [backgroundImage] is used. If that fails
+/// too, [backgroundColor] is considered.
```

Comment:
At line 20, I think being more explicit would be better since `that` is a bit vague in that context.

So instead of:
```
/// If [foregroundImage] fails then [backgroundImage] is used. If that fails
/// too, [backgroundColor] is considered.
```

it would be:
```
/// If [foregroundImage] fails then [backgroundImage] is used. If [backgroundImage] fails
/// too, [backgroundColor] is considered.
```